alphadia/transferlearning/train.py

```diff
@@ -315,6 +315,8 @@ def _test_ms2(
             The matched fragment intensity dataframe.
         metric_accumulator : MetricManager
             The metric manager object.
+        dataset : str
```

Comment:
maybe dataset is not precise enough. What do you think about `data_split` or `data_subset` instead? Ofc for the whole PR